framework/src/modules/chain/submodules/multisignatures.js

```diff
@@ -345,10 +341,7 @@ Multisignatures.prototype.getGroup = function(address, cb) {
 					if (!account) {
 						return setImmediate(
 							seriesCb,
-							new ApiError(
-								'Multisignature account not found',
-								errorCodes.NOT_FOUND
-							)
+							new Error('Multisignature account not found')
```

Comment:
Beware of this change. The error code could be propagated.

Reply:
This change was introduced to remove `ApiError` from Chain.  However this `getGroup` function is not used anywhere. If it were used, in HTTP API module `Multisignature account not found` would be caught and then returned in the JSON response with `NOT_FOUND` attached by using `ApiError` object. So, no problems as for now.